spec/serializers/basic_user_serializer_spec.rb

```diff
@@ -14,6 +14,16 @@
       expect(json[:avatar_template]).to eq(user.avatar_template)
     end
 
+    describe 'extended serializers' do
+      Fabricator(:post_action) do
```

Comment:
is this empty block needed ?

Reply:
No, it's not necessary. I'll clean that up!